suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -58,9 +58,11 @@ export const TransactionDetailHeader = ({
                 </Text>
                 <DiscreetText typography="titleMedium">{amount}</DiscreetText>
             </Box>
-            <DiscreetText typography="label" color="gray700">
-                {`≈  ${FiatAmountFormatter.format(fiatAmount ?? 0)}`}
-            </DiscreetText>
+            {fiatAmount && (
+                <DiscreetText typography="label" color="gray700">
+                    {`≈  ${FiatAmountFormatter.format(fiatAmount ?? 0)}`}
```

Comment:
no need for `?? 0` as fiatAmount won't be undefined | null in the formatter call